kimchi/src/circuits/polynomials/foreign_field_mul/witness.rs

```diff
@@ -135,28 +136,23 @@ fn compute_witness_variables<F: PrimeField>(
         carry0,
         carry1_lo,
         carry1_hi,
-        scaled_carry1_hi,
     ]
     .to_fields()
 }
 
 fn compute_bound_witness_variables<F: PrimeField>(
```

Comment:
perhaps a better name could be compute_bound_witness_carry now

Reply:
Also true.